### PR TITLE
feat: add trust registry endpoint

### LINKED ISSUE #139: Trustregistry endpoint
- [ ] Remove the current (wrong endpoint) providing the trust registry.
- [ ] Add a new endpoint that fetches the full trust registry from the trustregistry url under the default trustregistry route and provide it. This should be done because the Trustregistry is not (yet?) meant to be exposed to the general www but accessed only via the cloudapi app or from within the private network for now.

### PR DESCRIPTION
**Pull request by TimoGlastra**
Add a trust registry endpoint to the cloudapi.

Fixes #139

### CODE REVIEW

**Comment by TimoGlastra** [comment]
Thanks for fixing my codacy issues :)

File: app/generic/trust_registry.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+from fastapi import APIRouter
+
+import app.facades.trust_registry as trust_registry_facade
+
+logger = logging.getLogger(__name__)
+
+router = APIRouter(prefix="/trust-registry", tags=["trust registry"])
+
+
+@router.get("", response_model=trust_registry_facade.TrustRegistry)
```
Comment (morrieinmaas) [inline app/generic/trust_registry.py:11]: maybe `"/"` rather then `""`?

File: app/facades/trust_registry.py [hunk 2]
```diff
@@ -151,6 +157,16 @@ async def get_did_for_actor(actor_id: str) -> List[str]:
     return [did, didcomm_invitation]
 
 
+async def get_trust_registry() -> TrustRegistry:
+    print(TRUST_REGISTRY_URL)
```
Comment (morrieinmaas) [inline app/facades/trust_registry.py:161]: I'd say this `print` should either be log.info or similar or left out altogether

File: app/generic/trust_registry.py (hunk 1 above)
Comment (TimoGlastra) [reply app/generic/trust_registry.py:11]: Agreed

File: app/facades/trust_registry.py (hunk 2 above)
Comment (TimoGlastra) [reply app/facades/trust_registry.py:161]: Woops, shouldn't be there

File: trustregistry/models.py [hunk 3]
```diff
@@ -9,7 +9,7 @@ def schema_id_gen(context):
     did = parameters["did"]
     name = parameters["name"]
     version = parameters["version"]
-    return f"{did}:{name}:{version}"
+    return f"{did}:2:{name}:{version}"
```
Comment (morrieinmaas) [inline trustregistry/models.py:12]: lol, this `:2:` is gonna haunt me for the rest of eternity

File: trustregistry/registry/registry_actors.py [hunk 4]
```diff
@@ -39,7 +39,7 @@ async def get_actor_by_did(actor_did: str, db: Session = Depends(get_db)):
     return actor
 
 
-@router.delete("/{actor_id}")
+@router.delete("/{actor_id}", status_code=204)
```
Comment (morrieinmaas) [inline trustregistry/registry/registry_actors.py:42]: nice

**Comment by TimoGlastra** [comment]
Ignore my latest commit, added some code to the commit that shouldn't be there

**Review by morrieinmaas — COMMENTED** [review]
Nice job overall.

**Pull request merged**